<!DOCTYPE html>
<html><head>
	<meta name="generator" content="Hugo 0.76.5" />
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

    <title></title>
    <base href="https://jordiabante.github.io">
    <meta name="description" content="machine learning &amp; computational genomics">
    <meta name="author" content='Jordi Abante'>

    <link href="https://fonts.googleapis.com/css2?family=Inconsolata:wght@400;700&display=swap" rel="stylesheet">
    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">

    
    <link rel="stylesheet" href="/sass/researcher.min.css">

    

    
<script type="application/javascript">
var doNotTrack = false;
if (!doNotTrack) {
	(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
	(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
	m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
	})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
	ga('create', 'UA-180667417-1', 'auto');
	
	ga('send', 'pageview');
}
</script>


</head>

    <body><div class="container mt-5">
    <nav class="navbar navbar-expand-sm flex-column flex-sm-row p-0">
        <a class="navbar-brand mr-sm-auto" href="https://jordiabante.github.io">jordi abante</a>
        <div class="navbar-nav flex-row">
            
                
                
                    <a class="nav-item nav-link" href="/home">home</a>
                    
                        <span class="nav-item navbar-text mx-1">/</span>
                    
                
                    <a class="nav-item nav-link" href="/publications">publications</a>
                    
                        <span class="nav-item navbar-text mx-1">/</span>
                    
                
                    <a class="nav-item nav-link" href="/cv.pdf">cv</a>
                    
                        <span class="nav-item navbar-text mx-1">/</span>
                    
                
                    <a class="nav-item nav-link" href="/contact">contact</a>
                    
                
            
        </div>
    </nav>
</div>
<hr>
<div id="content">
<div class="container">
    <h2 id="about">about</h2>
<p><figure class="avatar">
    <img src="/avatar.png"/> 
</figure>

On May 10 of 2021, I successfully defended my Ph.D. dissertation entitled &ldquo;Statistical Signal Processing Methods for Epigenetic Landscape Analysis&rdquo; at the Electrical &amp; Computer Engineering department at Johns Hopkins University. In September 2021, I joined the Biomedical Data Science Department at Stanford University as a Postdoctoral Research Fellow, as well as the Stanford Center for Center for Computational, Evolutionary and Human Genomics.</p>
<h2 id="research-interests">research interests</h2>
<p>My main research interest lies at the intersection of statistical inference/machine learning and molecular biology.</p>
<h4 id="epigenetic-landscapes-phd">epigenetic landscapes (Ph.D.)</h4>
<p>During my Ph.D., I have been mostly concerned with statistical modeling 5mc DNA methylation. My work built upon that of Jenkinson et al. 2017 &amp; 2018, where the epigenetic landscape once envisioned by Conrad Waddington in the 1940s came to a reality for the first time. This first potential energy landscape accounted for the genome&rsquo;s architecture, and its application resulted in several novel biological findings. The seminal work in Jenkinson et al. 2017 &amp; 2018 laid the groundwork for the models and methods I developed during my doctoral studies. One of my last contributions during this phase was the development of a statistical method to study epigenetic landscapes from noisy nanopore sequencing data.</p>
<h4 id="epitranscriptomic-regulation-of-splicing-postdoc">epitranscriptomic regulation of splicing (postdoc)</h4>
<p>One of my current interests is to develop computational tools to study the epigenetic and epitranscriptomic regulation of alternative splicing, two fundamental biological questions closely interconnected, through sequencing data. This is a relatively new research avenue with the first direct demonstration that DNA methylation of exon-encoding regions alters exon inclusion levels occurring in 2018. Thus, we now know epigenetic marks play a role in alternative splicing through methylation (or lack of thereof) at exons or introns. We also know that epitranscriptomics is crucial in several cellular processes, such as proliferation, migration, or invasion. Indeed, epitranscriptomic marks are now known to play crucial context-specific roles in gene expression, and disruption of the epitranscriptomic mark m6A, for instance, is associated with altered patterns of mRNA cleavage and polyadenylation. Furthermore, there is also recent evidence of the roles of mRNA and noncoding RNA modiﬁcation in drug resistance, indicating that epitranscriptomic modifications could be the potential therapeutic targets in cancers. Nevertheless, there are still many critical gaps and challenges in the field.</p>
<p>Single-cell nanopore sequencing technology offers an excellent opportunity to study the interplay between epigenetics, epitranscriptomics, and alternative splicing. This technology has direct RNA sequencing capabilities, allows to sequence entire isoform at single-cell resolution, and captures information about both the genetic sequence itself as well as epigenetic or epitranscriptomic nucleotide modifications (e.g., m6A in RNA). By sequencing a combination of DNA, pre-mRNA, and mRNA at single-cell resolution, we can gain great insights into the processes behind splicing regulation. Moreover, one could experimentally validate the findings in vitro by applying a series of previously developed experimental techniques. Nevertheless, there are several computational challenges associated with the analysis of single-cell nanopore sequencing data, since it is noisy, sparse and has high dimensionality.</p>
<h4 id="mobilome-and-diversity-generating-mechanisms-postdoc">mobilome and diversity-generating mechanisms (postdoc)</h4>
<p>Current databases contain petabytes of metagenomic sequences, which are increasingly understood to impact human health dramatically. Indeed, the discovery of CRISPR-Cas systems has revolutionized biology, bioengineering, and biomedicine, as evidenced by the 2021 Nobel Prize in Chemistry. Digging into the history of CRISPR reveals that it was predicted to be a genome-modifying system as early as 2000 using purely computational methods. Given the diversity of microbial life on earth, there is every reason to believe that CRISPR is one of many mechanisms that diversify genomes and can be harnessed for biomedical applications one day. As a single motivating example, over 50% of bacteria are thought to lack CRISPR systems, suggesting that other molecular systems for genome modification exist in them.</p>
<p>Current computational approaches to detect CRISPR-like systems rely on heuristic assumptions and are not entirely statistical, limiting the breadth of discoveries that we can currently aspire to. My goal is to develop a new algorithmic paradigm that does away with heuristics and is grounded in classical statistics and machine learning, to discover alternative mechanisms to CRISPR, both in bacteria and viruses. This new paradigm will lead to groundbreaking discoveries with potentially transformative implications to genetics, cell biology, and medicine.</p>
<h2 id="bio">bio</h2>
<p>In 2014 I received my BS in Industrial Engineering from the Universitat Politècnica de Catalunya in Barcelona, where I majored in power electronics and signals. During my undergraduate, I served as the Team Captain of the ETSEIB Formula Student team, where I helped develop the first electric vehicle for the school. After graduating, I went on to obtain an MS from the Electrical &amp; Computer Engineering department at Texas A&amp;M University in College Station, Texas. There, I joined the Center for Bioinformatics and Genomic Systems Engineering, where I was involved in several computational genomics research projects under Dr. Aniruddha Datta. In 2015, I was awarded the “la Caixa” fellowship to pursue research in computational biology during my Ph.D. as a member of the Goutsias lab, developing computational methods to study epigenetic signatures in close collaboration with the Feinberg lab of the Johns Hopkins University School of Medicine. In addition to my research and Ph.D. coursework, I earned an MS focused on statistical learning from the Applied Mathematics &amp; Statistics department at Johns Hopkins University in 2018. After successfully defending my dissertation entitled &ldquo;Statistical Signal Processing Methods for Epigenetic Landscape Analysis&rdquo; on May 10 of 2021, I joined the Biomedical Data Science Department at Stanford University as a Postdoctoral Research Fellow being awarded the postdoctoral fellowship from the Center for Computational, Evolutionary and Human Genomics.</p>
<h3 id="references">References</h3>
<p>Available upon request.</p>

</div>

        </div>
    <div id="footer">
        <hr>
        <div class="container text-center mb-2">
            <ul class="social-icons">
                <a href="https://scholar.google.com/citations?user=VfbWX20AAAAJ"><img 
                src='logos/scholar.png' style="width:25px;height:25px;" /></a>
                <a href="http://www.github.com/jordiabante"><img src='logos/github.png' 
                style="width:25px;height:25px;" /></a>
                <a href="http://www.twitter.com/jordiatjhu"><img src='logos/twitter.png' 
                style="width:25px;height:25px;" /></a>
                <a href="http://www.linkedin.com/in/jordiabante"><img src='logos/linkedin.png' 
                style="width:25px;height:25px;" /></a>
                <a href="mailto:jordiabante@protonmail.com"><img src='logos/email.png' 
                style="width:25px;height:25px;" /></a>
            </ul>
        </div>
    </div>

</body>
</html>
