<!DOCTYPE html>
<html><head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

    <title></title>
    <base href="https://jordiabante.github.io">
    <meta name="description" content="machine learning &amp; computational genomics">
    <meta name="author" content='Jordi Abante'>

    <link href="https://fonts.googleapis.com/css2?family=Inconsolata:wght@400;700&display=swap" rel="stylesheet">
    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">

    
    <link rel="stylesheet" href="/sass/researcher.min.css">

    

    
<script type="application/javascript">
var doNotTrack = false;
if (!doNotTrack) {
	(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
	(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
	m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
	})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
	ga('create', 'UA-180667417-1', 'auto');
	
	ga('send', 'pageview');
}
</script>


</head>

    <body><div class="container mt-5">
    <nav class="navbar navbar-expand-sm flex-column flex-sm-row p-0">
        <a class="navbar-brand mr-sm-auto" href="https://jordiabante.github.io">jordi abante</a>
        <div class="navbar-nav flex-row">
            
                
                
                    <a class="nav-item nav-link" href="/home">home</a>
                    
                        <span class="nav-item navbar-text mx-1">/</span>
                    
                
                    <a class="nav-item nav-link" href="/publications">publications</a>
                    
                        <span class="nav-item navbar-text mx-1">/</span>
                    
                
                    <a class="nav-item nav-link" href="/cv.pdf">cv</a>
                    
                        <span class="nav-item navbar-text mx-1">/</span>
                    
                
                    <a class="nav-item nav-link" href="/contact">contact</a>
                    
                
            
        </div>
    </nav>
</div>
<hr>
<div id="content">
<div class="container">
    <h2 id="about">about</h2>
<p><figure class="avatar">
    <img src="/avatar.png"/> 
</figure>

On May 10 of 2021, I successfully defended my Ph.D. dissertation entitled &ldquo;Statistical Signal Processing Methods for Epigenetic Landscape Analysis&rdquo; at the Electrical &amp; Computer Engineering department at Johns Hopkins University. In September 2021, I joined the Biomedical Data Science Department at Stanford University as a Postdoctoral Research Fellow, as well as the Stanford Center for Center for Computational, Evolutionary and Human Genomics. In January of 2023, I joined the Department of Biomedical Sciences, as well as the Department of Mathematics and Computer Science at University of Barcelona.</p>
<h2 id="research-interests">research interests</h2>
<p>My main research interest lies at the intersection of statistical inference/machine learning and molecular biology.</p>
<h3 id="epigenetic-landscapes-phd--jhu">epigenetic landscapes (phd @ jhu)</h3>
<p>During my Ph.D., I have been mostly concerned with statistical modeling 5mc DNA methylation. My work built upon that of Jenkinson et al. 2017 &amp; 2018, where the epigenetic landscape once envisioned by Conrad Waddington in the 1940s came to a reality for the first time. This first potential energy landscape accounted for the genome&rsquo;s architecture, and its application resulted in several novel biological findings. The seminal work in Jenkinson et al. 2017 &amp; 2018 laid the groundwork for the models and methods I developed during my doctoral studies. One of my last contributions during this phase was the development of a statistical method to study epigenetic landscapes from noisy nanopore sequencing data.</p>
<h3 id="mobilome-and-diversity-generating-mechanisms-postdoc--stanford">mobilome and diversity-generating mechanisms (postdoc @ stanford)</h3>
<p>Current databases contain petabytes of metagenomic sequences, which are increasingly understood to impact human health dramatically. Indeed, the discovery of CRISPR-Cas systems has revolutionized biology, bioengineering, and biomedicine, as evidenced by the 2021 Nobel Prize in Chemistry. Digging into the history of CRISPR reveals that it was predicted to be a genome-modifying system as early as 2000 using purely computational methods. Given the diversity of microbial life on earth, there is every reason to believe that CRISPR is one of many mechanisms that diversify genomes and can be harnessed for biomedical applications one day. As a single motivating example, over 50% of bacteria are thought to lack CRISPR systems, suggesting that other molecular systems for genome modification exist in them.</p>
<p>Current computational approaches to detect CRISPR-like systems rely on heuristic assumptions and are not entirely statistical, limiting the breadth of discoveries that we can currently aspire to. My goal is to develop a new algorithmic paradigm that does away with heuristics and is grounded in classical statistics and machine learning, to discover alternative mechanisms to CRISPR, both in bacteria and viruses.</p>
<h2 id="bio">bio</h2>
<p>In 2014 I received my BS in Industrial Engineering (power electronics and signals) from the Universitat Politècnica de Catalunya in Barcelona. After graduating, I went on to obtain an MS from the Electrical &amp; Computer Engineering department at Texas A&amp;M University. There, I joined the Center for Bioinformatics and Genomic Systems Engineering, where I was involved in several computational genomics research projects (Datta lab). In 2015, I was awarded the “la Caixa” fellowship to pursue research in computational genomics during my Ph.D. as a member of the Goutsias lab, developing computational methods to study epigenetic signatures in close collaboration with the Feinberg lab of the Johns Hopkins University School of Medicine. In addition to my research and Ph.D. coursework, I earned an MS focused on statistical learning from the Applied Mathematics &amp; Statistics department at Johns Hopkins University in 2018. After successfully defending my dissertation entitled “Statistical Signal Processing Methods for Epigenetic Landscape Analysis” in May 2021, I joined the Biomedical Data Science Department at Stanford University (Salzman &amp; Ioannidis lab) as a Postdoctoral Research Fellow being awarded the the Stanford Center for Computational, Evolutionary and Human Genomics postdoctoral fellowship. In January 2023, I became a Postdoctoral Researcher in the Departments of Biomedical Sciences (Canals lab) and Mathematics and Computer Science at Universitat de Barcelona (Radeva lab) where I develop methods for multimodal single-cell data to study brain development and developmental alterations in Huntington’s disease.</p>
<h3 id="references">References</h3>
<p>Available upon request.</p>

</div>

        </div>
    <div id="footer">
        <hr>
        <div class="container text-center mb-2">
            <ul class="social-icons">
                <a href="https://scholar.google.com/citations?user=VfbWX20AAAAJ"><img 
                src='logos/scholar.png' style="width:25px;height:25px;" /></a>
                <a href="http://www.github.com/jordiabante"><img src='logos/github.png' 
                style="width:25px;height:25px;" /></a>
                <a href="http://www.twitter.com/jordiatjhu"><img src='logos/twitter.png' 
                style="width:25px;height:25px;" /></a>
                <a href="http://www.linkedin.com/in/jordiabante"><img src='logos/linkedin.png' 
                style="width:25px;height:25px;" /></a>
                <a href="mailto:jordiabante@protonmail.com"><img src='logos/email.png' 
                style="width:25px;height:25px;" /></a>
            </ul>
        </div>
    </div>

</body>
</html>
